src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -62,6 +62,10 @@
 
     <Exec Command="$(CoverageReportGeneratorCommandLine) -reports:$(CoverageOutputFilePath)"
           ContinueOnError="ErrorAndContinue" />
+
+    <Exec Command="start $(CoverageReportDir)index.htm"
+          Condition="'$(PopCoverageReport)' == 'true'" />
```

Comment:
Given it's an untyped language I'm leaning towards leaving it as a boolean so nobody using it someplace new could trip up and not realize there's two versions of true.